db/class.go

```diff
@@ -25,6 +25,8 @@ type Class struct {
 	Programs    []string `firestore:"programs" json:"programs"`
 	CID         string   `firestore:"CID" json:"cid"`
 	WID         string   `firestore:"WID" json:"wid"`
+	description string   `firestore:"description:"description"`
```

Comment:
```suggestion
	Description string   `firestore:"description:"description"`
```

Hey Anvi, I don't think I properly explained this but in Go. public variables are denoted by capitalizing the name of the variable.